src/openforms/formio/components/custom.py

```diff
@@ -162,7 +169,11 @@ def mutate_config_dynamically(
             # TODO: this should eventually be replaced with logic rules/variables that
             # retrieve data from an "arbitrary source", which will cause the data to
             # become available in the ``data`` argument instead.
-            child_choices = handler(bsn)
+            child_choices = handler(
+                bsn,
+                include_children=component["includeChildren"],
+                include_partners=component["includePartners"],
```

Comment:
perhaps `.get` is more appropriate, in the event that old forms are imported that didn't go through the data migration? Or we need to account for it during the import.

Reply:
`.get` with the defaults is better I think!